### PR TITLE
Added One plus to the whole

### PR DESCRIPTION
**Pull request by Nidhideveloper**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added One plus to the whole

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4210

### CODE REVIEW

**Comment by Nidhideveloper** [comment]
@HarshCasper Sir, plz check this and let me know if there is any problem with the code or not. thank u sir.

**Comment by rohanbabbar04** [comment]
@Nidhideveloper If you have an issue don't post here DM him the PR's which are not marked with the links

File: Java/other/OnePlustoWhole.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*
+
```
Comment (ankitaggarwal23) [inline Java/other/OnePlustoWhole.java]: Remove extra new line

File: Java/other/OnePlustoWhole.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+Enter the elements for array: [9, 9, 9, 9]
+Output--
+The incremented number is: [1, 0, 0, 0, 0]
+
+Example 3:
+Input--
+Enter the number of elements of array: 1
+Enter the elements for array: [0]
+Output--
+The incremented number is: [1]
```
Comment (ankitaggarwal23) [inline Java/other/OnePlustoWhole.java]: - Remove extra test case one is enough
- Add Space complexity

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by Nidhideveloper** [comment]
@ankitaggarwal23 Sir, I have made changes and please review it.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**